go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.8.4
 	go.uber.org/zap v1.24.0
-	golang.org/x/net v0.19.0
+	golang.org/x/net v0.23.0
```

Comment:
Please update DEPENDENCIES.md file to update the version of the following libs (no links to  https://clearlydefined.io/ are requested) 
- golang.org/x/net v0.23.0
- golang.org/x/sys v0.18.0
- golang.org/x/term v0.18.0